### PR TITLE
update credentials cors whitelist values

### PR DESCRIPTION
**Pull request by zubair-arbi**
ECOM-3572

@jimabramson @maxrothman @tobz

### CODE REVIEW

**Comment by jimabramson** [comment]
:+1:   @max whaddya think

**Comment by jimabramson** [comment]
@zubair-arbi actually does this need to be adjusted to escape the `/` char?   https://github.com/edx/configuration/blob/master/playbooks/roles/credentials/templates/edx/app/nginx/sites-available/credentials.j2#L20

**Comment by maxrothman** [comment]
:+1:, though I think a nearby comment explaining that the protocol is necessary would be nice.

File: playbooks/roles/credentials/defaults/main.yml [hunk 1]
```diff
@@ -125,7 +125,8 @@ CREDENTIALS_FILE_STORAGE_BACKEND:
   DEFAULT_FILE_STORAGE: 'django.core.files.storage.FileSystemStorage'
 
 CREDENTIALS_CORS_WHITELIST:
-  - "{{ CREDENTIALS_DOMAIN }}"
+  - "http://{{ CREDENTIALS_DOMAIN }}"
```
Comment (feanil) [inline playbooks/roles/credentials/defaults/main.yml:129]: Why allow `http`?

**Comment by jimabramson** [comment]
I've heard anecdotally that learners in some regions (CN?) access the site via http.  If that is not supported by actual evidence then I guess we can drop it.

**Comment by jimabramson** [comment]
@feanil ^ that was in response to your http/https question

**Comment by zubair-arbi** [comment]
@maxrothman I think its self explanatory since we have explicitly added the protocol.

**Comment by maxrothman** [comment]
@zubair-arbi the comment is to explain _why_ you added the protocol.

**Comment by maxrothman** [comment]
@jimabramson if you want to support both http and https, just add more entries to the map

**Comment by zubair-arbi** [comment]
@maxrothman I have added a comment. Please review

**Comment by maxrothman** [comment]
:+1:

**Pull request merged**